rest/model/distro.go

```diff
@@ -8,14 +8,16 @@ import (
 // APIDistro is the model to be returned by the API whenever distros are fetched.
 // EVG-1717 will implement the remainder of the distro model.
 type APIDistro struct {
-	Id APIString `json:"_id"`
+	Id           APIString `json:"_id"`
+	SpawnAllowed bool      `json:"spawn_allowed"`
```

Comment:
this should probably be "user_spawn_allowed" (we use "spawn" internally to distinguish static from dynamic hosts, so some extra clarity might be useful.

Reply:
in distro.Distro, does that SpawnAllowed field also refer to static vs dynamic hosts?

Reply:
No. Although no static hosts are spawnable, not all dynamic hosts are spawnable.